0_intro/exercises/hello_world/src/main.rs

```diff
@@ -1,3 +1,3 @@
 fn main() {
-    todo!("Print hello world on stdout! (not stderr)")
+    std::print!("hello world!\n");
```

Comment:
Those `print!` macros are among the few things that are included by default in your "namespace", see if you can call the macro directly.
It is usually preferred to call `println!` instead of `print!` if you intend to add a newline at the end.

Reply:
i thought it was a trap

Reply:
I see, I'll add a comment clarifying that issue, thanks !